sim-cli/src/main.rs

```diff
@@ -14,21 +17,6 @@ use sim_lib::{
 };
 use simple_logger::SimpleLogger;
 
-/// The default directory where the simulation files are stored and where the results will be written to.
-pub const DEFAULT_DATA_DIR: &str = ".";
-
-/// The default simulation file to be used by the simulator.
-pub const DEFAULT_SIM_FILE: &str = "sim.json";
-
-/// The default expected payment amount for the simulation, around ~$10 at the time of writing.
-pub const EXPECTED_PAYMENT_AMOUNT: u64 = 3_800_000;
-
-/// The number of times over each node in the network sends its total deployed capacity in a calendar month.
-pub const ACTIVITY_MULTIPLIER: f64 = 2.0;
-
-/// Default batch size to flush result data to disk
-const DEFAULT_PRINT_BATCH_SIZE: u32 = 500;
```

Comment:
Not sure about moving these defaults from in the codebase to into a config file. That moves us from being able to run `sim-ln` easily on its own to always needing a config file (or lots of flags) which makes things less usable (thinking about server contexts like docker where we'd now need to copy this config file into the container).

Reply:
I noted this in the changes I have introduced. The defaults remain, and allow us to handle the absence of a config file.